In out-of-tree builds the #include fails because $srcdir isn't in the include path.  Set CPPFLAGS so that it is.

Upstream-Status: Backport [r1750]
Signed-off-by: Ross Burton <ross.burton@intel.com>

Index: configure.ac
===================================================================
--- a/configure.ac	(revision 1749)
+++ b/configure.ac	(working copy)
@@ -159,6 +159,8 @@
 
 if test "$enable_jit" = "auto"; then
   AC_LANG(C)
+  SAVE_CPPFLAGS=$CPPFLAGS
+  CPPFLAGS=-I$srcdir
   AC_COMPILE_IFELSE([AC_LANG_SOURCE([[
   #define SLJIT_CONFIG_AUTO 1
   #include "sljit/sljitConfigInternal.h"
@@ -165,6 +167,7 @@
   #if (defined SLJIT_CONFIG_UNSUPPORTED && SLJIT_CONFIG_UNSUPPORTED)
   #error unsupported
   #endif]])], enable_jit=yes, enable_jit=no)
+  CPPFLAGS=$SAVE_CPPFLAGS
 fi
 
 # Handle --disable-pcregrep-jit (enabled by default)
